src/guides/v2.3/config-guide/prod/config-reference-systemxml.md

```diff
@@ -0,0 +1,359 @@
+
+### Tab node reference
+
+A `<tab>`-Tag can have the following child:
+
+| Node    | Description                                          | Type     |
+|---------|------------------------------------------------------|----------|
+| `label` | Defines the label that is displayed in the frontend. | `string` |
+
+### Example: Create a new tab
+
+The following code snippet demonstrates the creation of a new tab with example data.
+
+```xml
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="A_UNIQUE_ID" translate="label" class="a-custom-css-class-to-style-this-tab" sortOrder="10">
+            <label>A meaningful label</label>
+        </tab>
+    </system>
+</config>
+```
+
+The snippet above creates a new tab with the identifier `A_UNIQUE_ID`. As the `translate`-attribute is defined and references the label, the `label`-node is translatable. During the rendering process, the CSS class `a-custom-css-class-to-style-this-tab` will be applied on the HTML element that was created for this tab.
+The `sortOrder`-attribute with the value of `10` defines the position of the tab in the list of all tabs when rendered.
+
+## Sections
+
+A `<section>`-Tag references to either an existing or a new section in the system configuration.
+
+### Section attribute reference
+
+A `<section>`-Tag can have the following attributes:
+
+| Attribute       | Description                                                                                                                                         | Type     | Required |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|----------|----------|
+| `id`            | Defines the identifier that is used referencing the section.                                                                                        | `typeId` | required |
+| `translate`     | Defines the field that should be translatable. Provide `label` to make the label translatable.                                                      | `string` | optional |
+| `type`          | Defines the input type of the rendered HTML element. Defaults to `text`.                                                                            | `string` | optional |
+| `sortOrder`     | Defines the sort order of the section. High numbers will push the section to the bottom of the page; low numbers will push the section to the top. | `float`  | optional |
+| `showInDefault` | Defines whether the section is shown in the default store. Specify `1` to show the section and `0` to hide the section.                             | `int`    | optional |
```

Comment:
I would recommend clarifying this part. Technically, it's not a "default store" but "default scope" or "default config" 

![image](https://user-images.githubusercontent.com/2148959/79238064-33939100-7e6f-11ea-9848-22a1abdeee91.png)


```suggestion
| `showInDefault` | Defines whether the section is shown for the default configuration scope. Specify `1` to show the section and `0` to hide the section.                             | `int`    | optional |
```

Reply:
Makes sense! I guess this phrase appears a couple of times. I will do another interaction on it.